types/conf.go

```diff
@@ -179,6 +180,10 @@ func LoadNetConf(bytes []byte) (*NetConf, error) {
 		netconf.BinDir = defaultBinDir
 	}
 
+	if netconf.DefaultNetworkFile == "" {
+		netconf.DefaultNetworkFile = defaultDefaultNetworkFile
```

Comment:
@dougbtv how the delegate information is transferred from the default file to RawDelegates ?

Reply:
If we have defaultNetworkFile, then the delegate is not a must right ?